tests/widget/inline/widget-inline.webauthn-login.test.js

```diff
@@ -0,0 +1,38 @@
+import { expect, test } from '@playwright/test';
+import { asyncEvents, verifyUserInfo } from '../../utilities/async-events.js';
+
+test('inline widget with webauthn login', async ({ browser, page }) => {
+  const { clickButton, navigate } = asyncEvents(page);
+
+  let authenticator;
+  if (browser.browserType().name() === 'chromium') {
+    const cdpSession = await page.context().newCDPSession(page);
+
+    await cdpSession.send('WebAuthn.enable');
+    authenticator = await cdpSession.send('WebAuthn.addVirtualAuthenticator', {
+      options: {
+        protocol: 'ctap2',
+        transport: 'internal',
+        hasUserVerification: true,
+        isUserVerified: true,
+        hasResidentKey: true,
+      },
+    });
+
+    await navigate('widget/inline?journey=TEST_WebAuthn-Registration');
+
+    await page.getByLabel('Username').click();
+    await page.getByLabel('Username').fill('demouser');
+    await page.getByLabel('Username').press('Tab');
```

Comment:
Not sure why we are selecting the same element over and over. It's not a huge deal, but it is an asynchronous operation done 3 times in a row. If you need the same element for multiple actions, it's best to assign it to a local variable like this:

```
const username = page.getByLabel('Username');
username.click();
username.fill();
// ...
```

But, I'm also not sure why we are clicking on the element in the first place. Can't we just select it and fill it, no clicking or tabbing needed?

Reply:
done